.gitignore

```
@@ -17,11 +17,15 @@ values-init-done
 \#*\#
 
 # Envrc local overrides
-/.envrc.local
+.envrc.local
 
 # Nix-created result symlinks
 result
 result-*
+<<<<<<< HEAD
```

probably not supposed to be here.
```suggestion
<<<<<<< HEAD
```